notebooks/ch11/gradient_descent_define.md

```diff
@@ -30,340 +30,315 @@ tips['pcttip'] = tips['tip'] / tips['total_bill'] * 100
 
 ```python
 # HIDDEN
-def mse_cost(theta, y_vals):
+def mse(theta, y_vals):
     return np.mean((y_vals - theta) ** 2)
 
-def grad_mse_cost(theta, y_vals):
+def grad_mse(theta, y_vals):
     return -2 * np.mean(y_vals - theta)
 
-def plot_cost(y_vals, xlim, cost_fn):
+def plot_loss(y_vals, xlim, loss_fn):
     thetas = np.arange(xlim[0], xlim[1] + 0.01, 0.05)
-    costs = [cost_fn(theta, y_vals) for theta in thetas]
+    losses = [loss_fn(theta, y_vals) for theta in thetas]
     
     plt.figure(figsize=(5, 3))
-    plt.plot(thetas, costs, zorder=1)
+    plt.plot(thetas, losses, zorder=1)
     plt.xlim(*xlim)
-    plt.title(cost_fn.__name__)
-    plt.xlabel(r'$ \theta $')
-    plt.ylabel('Cost')
+    plt.title(loss_fn.__name__)
+    plt.xlabel(r'$ \hat{\theta} $')
+    plt.ylabel('Loss')
     plt.legend()
     
-def plot_theta_on_cost(y_vals, theta, cost_fn, **kwargs):
-    cost = cost_fn(theta, y_vals)
+def plot_theta_on_loss(y_vals, theta, loss_fn, **kwargs):
+    loss = loss_fn(theta, y_vals)
     default_args = dict(label=r'$ \theta $', zorder=2,
                         s=200, c=sns.xkcd_rgb['green'])
-    plt.scatter([theta], [cost], **{**default_args, **kwargs})
+    plt.scatter([theta], [loss], **{**default_args, **kwargs})
 
-def plot_tangent_on_cost(y_vals, theta, cost_fn, eps=1e-6):
-    slope = ((cost_fn(theta + eps, y_vals) - cost_fn(theta - eps, y_vals))
+def plot_tangent_on_loss(y_vals, theta, loss_fn, eps=1e-6):
+    slope = ((loss_fn(theta + eps, y_vals) - loss_fn(theta - eps, y_vals))
              / (2 * eps))
     xs = np.arange(theta - 1, theta + 1, 0.05)
-    ys = cost_fn(theta, y_vals) + slope * (xs - theta)
+    ys = loss_fn(theta, y_vals) + slope * (xs - theta)
     plt.plot(xs, ys, zorder=3, c=sns.xkcd_rgb['green'], linestyle='--')
 ```
 
 ## Gradient Descent
 
-We are interested in creating a function that can minimize a cost function without forcing the user to predetermine which values of $ \theta $ to try. In other words, while the `simple_minimize` function has the following signature:
+We are interested in creating a function that can minimize a loss function without forcing the user to predetermine which values of $ \theta $ to try. In other words, while the `simple_minimize` function has the following signature:
```

Comment:
without forcing the user to predetermine which values of θ -> theta to \hat{theta_0}